kuma/wiki/models.py

```diff
@@ -1714,7 +1714,7 @@ def get_section_content(self, section_id):
         """Convenience method to extract the content for a single section"""
         return self.document.extract_section(self.content, section_id)
 
-    def get_tidied_content(self):
+    def get_tidied_content(self, allow_blank=False):
```

Comment:
docstring could use update for new functionality (and to remove that trailing comma after ``field,`` :grimacing: